### PR TITLE
L4C-3D and Centaur combination 

### PR DESCRIPTION
**Pull request by CBurton90**

### CODE REVIEW

**Review by salichon — COMMENTED** [review]
Kia Ora! thanks for this
I d say we don't need the L4C to be included in that PR 
They are no longer used since the late 2000 ... with  of course ... :) the
exception of ...
`Sercel Inc. | L4C | WTAZ_AKN | WTAZ | 10 | 0 |   | 0 | 5 | 0 | 0 | 0 | 0 | 1995-01-10T16:30:00Z | 9999-01-01T00:00:00Z`
mounted with a Q330
So we can inlude this almost retired instrument in our PR or not. (depending of its planned future for the next decade :))
Up to the `network dev` maybe ? 
Cheers! 
(and L4C  are one component afaik)

File: resp/responses/configurations/shortperiod.yaml [hunk 1]
```diff
@@ -144,6 +144,18 @@ response:
 
   Nanometrics Dataloggers Connected to Short Period Sensors:
     sensors:
+    - sensors:
+      - L4C
```
Comment (ozym) [inline resp/responses/configurations/shortperiod.yaml:148]: Unless we plan on plugging a 1D sensor into a centaur, I'd not bother with this section.

**Review by ozym — CHANGES_REQUESTED** [review]

File: resp/responses/configurations/shortperiod.yaml (hunk 1 above)
Comment (ozym) [reply resp/responses/configurations/shortperiod.yaml:148]: Ah, just spotted @salichon bet me too the punch!

Comment (CBurton90) [reply resp/responses/configurations/shortperiod.yaml]: @salichon @ozym consider it gone!

**Comment by ozym** [comment]
still needs the `auto.go` to be updated.

**Review by ozym — APPROVED** [review]

**Comment by salichon** [comment]
will approve and merge afetra stationxml  generation locally  :)

**Review by salichon — APPROVED** [review]
BSWZ CTR4 + LC4-3D Paired into the xml  
great!

**Pull request merged**